### PR TITLE
Make the body text use helvetica.

### PR DESCRIPTION
**Pull request by markstory**
A few people have mentioned that they find Raleway tiring to read for longer stretches. Helvetica is a bit more battle-tested as a body typeface, and is generally less abrasive to folks.

If people are happy with this, I'll port this change to the other book branches.

screen shot 2016-06-28 at 21 56 04

### CODE REVIEW

**Comment by ionas** [comment]
Helvetica is quite overused. It also does not offer a monospaced version to go along with it.
I ll repeat myself on this:
1. **Github's Font Stack**: `"Helvetica Neue", Helvetica, "Segoe UI", Arial, freesans, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"`  
^ Coders are used to this and will feel at home. The sheer scale of GH means it is tested across multiple devices, setups etc. See the argument for Droid Sans below. It also supports emoticons (say for future use on `discourse.cakephp.org`).
2. **Open Sans** https://typekit.com/fonts/open-sans/n4/wfs ("Open Sans",Helvetica,Arial,sans-serif)  
^ IMHO renders most beautiful for a FLOSS sans-serif and does not tire your eyes.
3. **Source Sans Pro** https://typekit.com/fonts/source-sans-pro/n4/wfs  
^ Great cause we can couple it with Source Code Pro
4. **Droid Sans** https://typekit.com/fonts/droid-sans/n4/wfs  
^ Consider it because it is made to run on all kind of different devices and in all kind of different conditions
https://github.com/cakephp/docs/issues/4012#issuecomment-222433460

Open Sans or Source Code Pro etc are used on MySQL Docs, Babelfish Docs, Laravel Docs etc. I don't understand why we don't pick up something like that over Helvetica.

Anyway, despite that monospaced and non-monospaced won't map nicely in case Helvetica is picked, it is still an improvement over Raleway for floating-text.

File: themes/cakephp/static/css/default.css [hunk 1]
```diff
@@ -28,6 +29,13 @@
     margin: -50px 0 0; /* negative fixed header height */
 }
 
+.document-body {
+    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-weight: normal;
+    font-size: 15px;
+    letter-spacing: 0.02em;
```
Comment (ionas) [inline themes/cakephp/static/css/default.css]: Letter spacing is not a good idea on floating text as it undos well-kerned font rendering.
https://developer.mozilla.org/de/docs/Web/CSS/font-kerning

**Comment by markstory** [comment]
Helvetica was recommended by the designer of the new designs. I'll compare what the output looks like with letter-spacing off though.

**Comment by htstudios** [comment]
Thank you. Helvetica at standard weight and 'normal' font sized should not need any on flosting/body/paragraph text.

I don't have contact with the designer but maybe he wants to take a look st the 3 stacks suggested above for the docs/source code/api.

**Comment by lorenzo** [comment]
I'm going to merge this as it is an improvement... The letter spacing stuff can be tweaked later as well

**Pull request merged**